armtemplates/parameters.bicepparam

```diff
@@ -0,0 +1,16 @@
+using 'template.bicep' /*TODO: Provide a path to a bicep template*/
+
+param virtualMachines_iagora_name = 'iagora'
+
+param publicIPAddresses_iagora_ip_name = 'iagora-ip'
+
+param virtualNetworks_iagora_vnet_name = 'iagora-vnet'
+
+param networkInterfaces_iagora731_z1_name = 'iagora731_z1'
+
+param networkSecurityGroups_iagora_nsg_name = 'iagora-nsg'
```

Comment:
Instead of configuring the name of each resource individually, what about using a single `name_prefix` configuration that would act like a namespace? For example, if `name_prefix` is set to "iagora" then we could name the VM resource `'${name_prefix}-vm'`, the IP resource `'${name_prefix}-ip'` etc.
WDYT?

Reply:
UPD: Most examples on Microsoft Learn seem to use camelCase to name parameters. Thus, the name prefix would be `namePrefix`.